### PR TITLE
post: ng and cli v15.1

### PR DESCRIPTION
**Pull request by cexbrayat**

### CODE REVIEW

File: _posts/2023-01-09-what-is-new-angular-15.1.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+## Angular CLI
+
+As usual, you can check out our dedicated article about the new CLI version:
+
+👉 [Angular CLI v15.1](/2023/01/09/angular-cli-15.1)
+
+
+## Summary
+
+This release is packed with features as you can see,
```
Comment (jnizet) [inline _posts/2023-01-09-what-is-new-angular-15.1.md:70]: I wouldn't say it's packed with features, no.

**Review by jnizet — CHANGES_REQUESTED** [review]

File: _posts/2023-01-09-what-is-new-angular-15.1.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+`CanLoad` was also only running once, whereas `CanMatch` runs on every navigation
+as the `CanActivate` guard.
+
+It is now possible to define the `onSameUrlNavigation` option for a specific navigation to specify what to do when the user navigates to the same URL as the current one,
+with two possible values: `reload` and `ignore`.
+This was previously only possible globally with the `RouterConfigOptions` of the router (or `withRouterConfig` if you're using the standalone router providers).
+
+You can now do something like:
+
+    this.router.navigateByUrl('/user', { onSameUrlNavigation: 'reload' })
```
[the remark this answers is not in the document]
Comment (cexbrayat) [reply _posts/2023-01-09-what-is-new-angular-15.1.md:38]: IIRC it recreates the component if you combine that with a reuse strategy

File: _posts/2023-01-09-angular-cli-15.1.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+You can use:
+
+    ng generate config karma
+
+to generate the `karma.conf.js` file, and:
+
+    ng generate config browserslist
+
+to generate the `.browserslistrc` file.
```
[the remark this answers is not in the document]
Comment (cexbrayat) [reply _posts/2023-01-09-angular-cli-15.1.md:72]: Nope ^^

File: _posts/2023-01-09-angular-cli-15.1.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Let's see what we've got in this release.
+
+## esbuild builder improvements
+
+The new experimental builder that uses [esbuild](https://esbuild.github.io/)
+has been introduced in v14, but with some missing features.
+
+Even if it is still not feature complete, the esbuild builder is making progress.
+It now supports Sass inline styles. Less is not supported yet.
```
[the remark this answers is not in the document]
Comment (cexbrayat) [reply _posts/2023-01-09-angular-cli-15.1.md:23]: When it's directly in the component decorator?

File: _posts/2023-01-09-what-is-new-angular-15.1.md (hunk 1 above)
Comment (cexbrayat) [reply _posts/2023-01-09-what-is-new-angular-15.1.md:70]: Good catch, it's a leftover my previous blog post

**Pull request merged**